src/config/mod.rs

```diff
@@ -56,6 +59,15 @@ impl Config {
             .flat_map(|ig| ig.patterns)
             .collect::<Vec<_>>();
 
+        let index_extra = match index_extra {
+            Some(f) => f,
+            None => {
+                return Err(crate::Error::ConfigFileParseError {
+                    error: "index-extra section not found".to_string(),
+                })
```

Comment:
This is wrong, this is an optional section. Use empty string if its not found. Or if you want to give error give only case backend is set to raw, in ftd backend we do not need index-extra for example.